evansmusomi/burger-builder
# Adds Order Summary with Options to Cancel or Continue

*What this does*
- Adds order summary modal
- Adds backdrop when modal is displayed
- Adds custom button

*How to verify*
- User can see order summary modal when they select "ORDER NOW"
- User can see a black, semi-transparent backdrop when the order summary is displayed
- User can see and interact with custom cancel and continue buttons on order summary
